.travis.yml

```diff
@@ -0,0 +1,24 @@
+os: osx
+sudo: required
+osx_image: xcode6.4
+language: generic
+install:
+- brew update
+- brew upgrade libtool automake autoconf
+- ./autogen.sh
+- autoreconf -i
+- ./configure
```

Comment:
Is there a reason to call `autogen.sh` / `autoreconf` / `configure` if `cmake` is used ?